README.md

```diff
@@ -1,2 +1,24 @@
 ## Working with Git
-Please see our Git Standard Operating Procedures in our documents repository.
\ No newline at end of file
+Please see our Git Standard Operating Procedures in our documents repository.
+
```

Comment:
Let's pull the items into different sections.

# Title
description

## Table of Contents

## Installation

## Local Development
- We can put in the info about the different scripts here

## Working with Git
- Link to the documents repo with Git SOP
- Let's have the instructions for command line point to using the terminal directly and not to the VSC stuff to get folks used to working with the command line directly and not relying on the IDE.